jme3-core/nbproject/project.xml

```diff
@@ -24,17 +24,11 @@
                         <specification-version>3.3.0</specification-version>
                     </run-dependency>
                 </dependency>
-                <dependency>
-                    <code-name-base>com.jme3.gde.core.updatecenters</code-name-base>
-                    <run-dependency>
-                        <specification-version>3.3.0</specification-version>
-                    </run-dependency>
-                </dependency>
```

Comment:
I am worried by this: Why and how does jme3-core depend on the update centers?

Reply:
Maybe just bundle it to the project? How could this be verified? There is no direct dependency at least. And there shouldn't either.

Reply:
Probably only by testing everything and seeing if one is getting crashes.
Additionally maybe searching through the relevant code.

Specifically I think about some code in the Installer.java files, which are executed on startup, which most certainly add the update centers.